python/cudf/cudf/core/index.py

```diff
@@ -1423,24 +1479,80 @@ def get_slice_bound(self, label, side, kind=None):
         return self._values.get_slice_bound(label, side, kind)
 
     def is_numeric(self):
+        warnings.warn(
```

Comment:
Can we not avoid redefining these methods in subclasses of `GenericIndex`? They should just work by inheritance.

Reply:
Yup, we don't need the duplication here. Dropped them 👍